spec/unit/puppet/provider/host/augeas_spec.rb

```diff
@@ -273,9 +273,9 @@
       # Change Puppet::Type::Host.validattr? to return false instead for
       # comment so it throws the same errors as Puppet < 2.7
       validattr = Puppet::Type.type(:host).method(:validattr?)
-      Puppet::Type.type(:host).stubs(:validattr?).with { |arg| validattr.call(arg) }.returns(true)
-      Puppet::Type.type(:host).stubs(:validattr?).with { |arg| ! validattr.call(arg) }.returns(false)
-      Puppet::Type.type(:host).stubs(:validattr?).with(:comment).returns(false)
+      allow(Puppet::Type.type(:host)).to receive(:validattr?).with { |arg| validattr.call(arg) }.and_return(true)
+      allow(Puppet::Type.type(:host)).to receive(:validattr?).with { |arg| ! validattr.call(arg) }.and_return(false)
```

Comment:
This is invalid, but I wonder if this should be
```suggestion
      allow(Puppet::Type.type(:host)).to receive(:validattr?).and_call_original
```

On the other hand, we could probably also drop Puppet < 2.7 testing.